daemon/logger/journald/read.go

```diff
@@ -234,25 +234,24 @@ drain:
 			break
 		}
 	}
-	retCursor := ""
-	if C.sd_journal_get_cursor(j, &cursor) == 0 {
-		retCursor = C.GoString(cursor)
-		C.free(unsafe.Pointer(cursor))
-	}
-	return retCursor
+
+	// free(NULL) is save
```

Comment:
it's funny because the line just below says 'unsafe'... I will fix it in the next version.